libenkf/src/rng_manager.c

```diff
@@ -104,6 +124,10 @@ rng_type * rng_manager_iget(rng_manager_type * rng_manager, int index) {
 }
 
 
+void rng_manager_get_state(const rng_manager_type * rng_manager, char * state) {
```

Comment:
Maybe `rng_manager_export_state( )` is better - since the function actually is `void`! But see other comment in `rng_config.c` - unless I have misunderstood something else I would actually recommend the complete removal of this function.